src/main/java/uk/gov/cshr/report/repository/DbRepository.java

```diff
@@ -46,11 +46,11 @@ public DbRepository(DataSource dataSource) {
     }
 
     public List<ModuleRecord> getModuleRecords(LocalDate from, LocalDate to) {
```

Comment:
Good to add tests where possible,.

Reply:
To properly test the implemented fix, integration tests with local database would be required. Unit tests with JdbcTemplate mock would not show that the fix works properly. However the manual tests were performed on both performance and staging environment which prove that the fix works as intended.

Reply:
This requires deployment to an environment with a real DB to test because it's on DB repository level and only invoked when hitting the DB, until such a time as LPG-wide e2e tests are included.